OfficeUIFabric/Controls/MSLabel.swift

```diff
@@ -14,6 +14,11 @@ open class MSLabel: UILabel {
             update()
         }
     }
+    @objc open var maxPointSize: CGFloat = .greatestFiniteMagnitude {
```

Comment:
Change default value to 0 - this is what you actually use in code below (and it makes more sense and a more accepted practice).

Reply:
If we choose to use 0 as a "magic" number where 0 indicates that there is no maximum point size, can we put a comment making that behavior explicit?